plugins/versioning/src/main/resources/dokka/scripts/version-navigation.js

```diff
@@ -0,0 +1,40 @@
+window.addEventListener('load', () => {
+    document.querySelectorAll(".versions-dropdown a")
+            .forEach(elem => elem.addEventListener('click', (event) => selectVersion(event, elem)))
+})
+
+const getLastAvailableVersion = () => {
+    const versions = document.querySelector(".versions-dropdown-data").children
+    for (let i = versions.length - 1; i >= 0; i--) {
+        let element = versions[i]
+        if(!element.classList.contains("unavailable-version")) {
+            return element
+        }
+    }
+}
+
+const selectVersion = (event, elem) => {
+    if(elem.classList.contains("unavailable-version")) {
+        event.preventDefault()
+        document.querySelector(".versions-dropdown-button").textContent = elem.textContent
+
+        let lastVersion = getLastAvailableVersion()
+        const appended = document.createElement("iframe")
+        appended.src = elem.getAttribute("href")
+        appended.frameBorder = 0
+        appended.height="100%"
+        appended.onload = function() {
+            const messageBox = appended.contentWindow.document.querySelector(".sub-title")
+            messageBox.textContent="The declaration is unavaibaile in " +
+            elem.textContent + " version, but this exists in "
+            let link = appended.contentWindow.document.createElement("a")
+            link.href = lastVersion.href
+            link.textContent =  lastVersion.textContent
+            link.target = "_parent"
+            messageBox.append(link)
+        };
```

Comment:
If you have stdlib built locally with these changes, could you please publish it somewhere? 

I wanted to check the following cases:

1. Does it always lead to the latest version?  Case: I have a declaration in 1.1, which was removed in 1.2, and the latest version is 1.3. What happens if I go to version 1.1, choose the removed declaration and select 1.2? Will it tell me it exists in 1.3?
2. If you click on the proposed latest version, will it load the whole page or only the main content in the middle? In other words, you will see navigation from which version, initial or proposed?

Reply:
https://vmishenev.github.io/kotlin-dokka-stdlib/prefinal/

Reply:
Not sure how the linked docs were built - it it a bug of the versioning plugin or of configuration? 

`decapitalize` was not deprecated in 1.0

![2022-11-26_06-12-19](https://user-images.githubusercontent.com/22685910/204074953-f913b8c7-ace5-40db-8749-b32952e82a12.png)

Reply:
Also not sure how this happened, but `isISOcontrol` exists in 1.0 and 1.5-1.7. 

If you open it for 1.8, then choose 1.4, it'll say that it exists in 1.0, which is true, but I'd say it's less useful then if it pointed to the latest version

![2022-11-26_07-18-09](https://user-images.githubusercontent.com/22685910/204075187-fe211165-5432-4f80-a1b6-b248d0d5e704.png)

Reply:
Navigation can also be confusing. You go to 1.8 and click on `isISOControl`, then choose version 1.1, it changes the dropdown on top to 1.1, but if you click anywhere in the navigation it will lead to 1.8

I reckon it should either not change the version on top, or re-draw navigation menu to fit the newly selected version

Reply:
> Not sure how the linked docs were built - it it a bug of the versioning plugin or of configuration?

None of them. It can be done in the filter plugin, but now it works like old Dokka: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/decapitalize.html
Anyway, we should display the version of Kotlin for `DeprecatedSinceKotlin`.

Reply:
> Also not sure how this happened, but isISOcontrol exists in 1.0 and 1.5 - 1.7 only.

It is a bug in the filter plugin and not relevant to this PR. I will fix it after merging.